Update clinic-user-guide-for-tiup.md (#10925)
pingcap/docs: clinic/clinic-user-guide-for-tiup.md

    > **Note:**
    >

>     > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md) to do the pre-steps.

Reviewer: ```suggestion
    > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#Prerequisites) and complete the prerequisites.
```

>     > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#Prerequisites) and complete the prerequisites.

Reviewer: ```suggestion
    > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#prerequisites) and complete the prerequisites.
```

>     > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#prerequisites) and complete the prerequisites.

Reviewer 2: ```suggestion
    > - When accessing Clinic Server for the first time, before getting a token, you need to prepare the environment by referring to [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#prerequisites).
```
